org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/StepModel.java

```diff
@@ -31,18 +32,31 @@
 	@FieldDescriptor(label="Step", scannable="name", hint="This is the step during the scan", fieldPosition=3) // The scannable lookup gets the units
 	private double step;
 	
+	@FieldDescriptor(label="Exposure Time", 
+			         hint="If you set this field the exposure times for all\ndetectors in the scan will be changed when the first\npoint of the step is moved.", 
+			         unit="s", 
+			         minimum=0.001, 
+			         fieldPosition=4,
+			         role=FieldRole.EXPERT) // TODO This is optional
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)